### PR TITLE
refactor: removed unused tsconfig.build.json

### PR DESCRIPTION
**Pull request by andykenward**
Removed the the tsconfig.build.json configuration files as it appears they arent being used in the project.

### CODE REVIEW

**Review by peduarte — APPROVED** [review]
That's awesome. I really love these types of refactoring. Thanks!

File: packages/react/tsconfig.json [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["src/**/*", "tests/**/*"]
+  "compilerOptions": {
+    "jsx": "react",
+  },
+  "include": ["src/**/*"]
```
Comment (peduarte) [inline packages/react/tsconfig.json:6]: just one question: why did you remove tests from here?

Comment (andykenward) [reply packages/react/tsconfig.json:6]: As the project is using Jest along with ts-jest to transform the TypeScript files which gives Jest allows typechecking to work.
Without ts-jest then you wouldn't get typechecking when running the tests as Jest uses Babel for TypeScript transforming.

But we should probably add `"test/**/*"` back in the `include` array for both `ts.config.json` in the packages folders incase you want to use TypeScript directly via the cli `yarn typecheck`.

Comment (peduarte) [reply packages/react/tsconfig.json:6]: @andykenward gotcha. so I'll wait for you to add it back before merging. Cheers!

**Review by peduarte — APPROVED** [review]

File: packages/react/tsconfig.json (hunk 1 above)
Comment (andykenward) [reply packages/react/tsconfig.json]: @peduarte Done.

But if the project ever changes the build setup it may need removing.

**Review by hadihallak — APPROVED** [review]

**Comment by hadihallak** [comment]
Thanks @andykenward 
loving the refactors
